tests/main.cpp

```diff
@@ -68,14 +68,14 @@ bool testCrypto(QRSAEncryption::Rsa keyLength) {
 
             encodeData += "work it";
 
-            if (e.checkSignMessage(encodeData, pub)) {
+            if (e.checkSignMessage(encodeData, pub, keyLength)) {
                 qCritical() << "sig message error RSA with added value to back" << keyLength;
                 return false;
             }
 
             encodeData.push_front("not work");
 
-            if (e.checkSignMessage(encodeData, pub)) {
+            if (e.checkSignMessage(encodeData, pub, keyLength)) {
```

Comment:
By project, this parameter is called rsa and not keyLength
replace keyLength to rsa

Reply:
this still not fixed